app/views/home/index.html.erb

```diff
@@ -7,18 +7,13 @@
 
 		<!--Welcome section-->
         <section class="waypoint video-block fullscreen" data-hash="welcome">
-			<%= video_tag(
-				"home/woman-small.mov",
-				id: "womanvideo",
-				autoplay: "autoplay",
-				muted: "muted",
-				loop: "loop",
-				poster: "home/woman_front.jpg",
-				class: "background-video",
-				"x-webkit-airplay": true,
-				"webkit-playsinline": true,
-				preload: "auto"
-			) %>
+
+			<video id="womanvideo" autoplay="autoplay" muted="muted" loop="loop" poster    ="assets/home/woman_front.jpg" 
+			 	    class="background-video" x-webkit-airplay="true" webkit-playsinline="true" preload="auto">
+					<source data-src="assets/home/woman-small.mov" type="video/mp4">
+					<source data-src="assets/home/woman-small.ogm" type="video/ogm">
+			</video>
+
```

Comment:
Bo, can you specify your concerns on this? Is it related to performance or is it related to not working completely? 
As performance concern, I see that we introduce "cache invalidation", but I do see this is not big performance impact. 
If you say it is not working completely, then that's what I don't quite understand.